client/my-sites/site-settings/form-seo.jsx

```diff
@@ -400,15 +404,58 @@ export const SeoForm = React.createClass( {
 					) }
 				</Card>
 
+				{ ! showAdvancedSeo &&
+					<Card className="seo-form-nudge">
+						<div className="seo-form-nudge__business-plan-card">
+							<PlanCompareCard
+								title={ getPlan( PLAN_BUSINESS ).getTitle() }
```

Comment:
Note that the above is a blocker if this nudge is being shown outside of an abtest.